virtcontainers/qemu.go

```diff
@@ -931,6 +931,17 @@ func (q *qemu) waitSandbox(timeout int) error {
 	return nil
 }
 
+func isConnectionRefusedError(err error) bool {
+	if neterr, ok := err.(*net.OpError); ok {
```

Comment:
If `err` is `nil` this will panic I think.

Reply:
Hm, true. Though currently it'll be only called with non-nill values. Anyway, I'll just add another check here.